.github/CODEOWNERS

```diff
@@ -233,7 +233,7 @@ receiver/googlecloudspannerreceiver/              @open-telemetry/collector-cont
 receiver/haproxyreceiver/                         @open-telemetry/collector-contrib-approvers @atoulme @MovieStoreGuy
 receiver/hostmetricsreceiver/                     @open-telemetry/collector-contrib-approvers @dmitryax @braydonk
 receiver/httpcheckreceiver/                       @open-telemetry/collector-contrib-approvers @codeboten
-receiver/huaweicloudcesreceiver/                  @open-telemetry/collector-contrib-approvers @narcis96
+receiver/huaweicloudcesreceiver/                  @open-telemetry/collector-contrib-approvers @heitorganzeli @narcis96 @mwear
```

Comment:
I'm not sure how this got in, but #36258 seems to have introduced all three code owners, but this file got only one. Generating the code owners file again appears to have caught this difference.